Can't install dependencies using Node 12

The library can be compiled using Node 10 (end of life 2021-04-30) though gives error using Node 12.

The problem seems to be related with the `keythereum` dependency. See log:

```
alastria-identity-lib$ git checkout origin/develop && git clean -dxf && nvm use 12 && npm i

HEAD está ahora en 3af6b2c Merge pull request #114 from alastria/bug/ether.js_version
Borrando node_modules/
Now using node v12.18.3 (npm v6.14.6)

> scrypt@6.0.3 preinstall /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/scrypt
> node node-scrypt-preinstall.js


> keccak@3.0.1 install /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/keccak
> node-gyp-build || exit 0


> keccak@1.4.0 install /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/keythereum/node_modules/keccak
> npm run rebuild || echo "Keccak bindings compilation fail. Pure JS implementation will be used."


> keccak@1.4.0 rebuild /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/keythereum/node_modules/keccak
> node-gyp rebuild

make: se entra en el directorio '/home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/keythereum/node_modules/keccak/build'
  CXX(target) Release/obj.target/keccak/src/addon.o
../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::Initialize(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/addon.cc:37:47: error: no matching function for call to ‘v8::Value::IntegerValue()’
     unsigned int rate = info[0]->IntegerValue();
                                               ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../src/addon.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note: candidate: v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
                                        ^~~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
../src/addon.cc:38:51: error: no matching function for call to ‘v8::Value::IntegerValue()’
     unsigned int capacity = info[1]->IntegerValue();
                                                   ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../src/addon.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note: candidate: v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
                                        ^~~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::AbsorbLastFewBits(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/addon.cc:56:48: error: no matching function for call to ‘v8::Value::IntegerValue()’
     unsigned char bits = info[0]->IntegerValue();
                                                ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../src/addon.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note: candidate: v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
                                        ^~~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::Squeeze(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/addon.cc:64:43: error: no matching function for call to ‘v8::Value::IntegerValue()’
     size_t length = info[0]->IntegerValue();
                                           ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../src/addon.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note: candidate: v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
                                        ^~~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
../src/addon.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE KeccakWrapper::Copy(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/addon.cc:75:82: error: no matching function for call to ‘v8::Value::ToObject()’
     KeccakWrapper* to = Nan::ObjectWrap::Unwrap<KeccakWrapper>(info[0]->ToObject());
                                                                                  ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../src/addon.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2672:44: note: candidate: v8::MaybeLocal<v8::Object> v8::Value::ToObject(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT MaybeLocal<Object> ToObject(
                                            ^~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2672:44: note:   candidate expects 1 argument, 0 provided
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../src/addon.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2686:31: note: candidate: v8::Local<v8::Object> v8::Value::ToObject(v8::Isolate*) const
                 Local<Object> ToObject(Isolate* isolate) const);
                               ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2686:31: note:   candidate expects 1 argument, 0 provided
                 Local<Object> ToObject(Isolate* isolate) const);
                               ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
keccak.target.mk:129: fallo en las instrucciones para el objetivo 'Release/obj.target/keccak/src/addon.o'
make: *** [Release/obj.target/keccak/src/addon.o] Error 1
make: se sale del directorio '/home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/keythereum/node_modules/keccak/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/rbarriuso/.nvm/versions/node/v12.18.3/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Linux 4.15.0-121-generic
gyp ERR! command "/home/rbarriuso/.nvm/versions/node/v12.18.3/bin/node" "/home/rbarriuso/.nvm/versions/node/v12.18.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/keythereum/node_modules/keccak
gyp ERR! node -v v12.18.3
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! keccak@1.4.0 rebuild: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the keccak@1.4.0 rebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rbarriuso/.npm/_logs/2020-10-16T16_26_00_059Z-debug.log
Keccak bindings compilation fail. Pure JS implementation will be used.

> scrypt@6.0.3 install /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/scrypt
> node-gyp rebuild

make: se entra en el directorio '/home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/scrypt/build'
  SOLINK_MODULE(target) Release/obj.target/copied_files.node
  COPY Release/copied_files.node
  CC(target) Release/obj.target/scrypt_wrapper/src/util/memlimit.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/keyderivation.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/pickparams.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/hash.o
  AR(target) Release/obj.target/scrypt_wrapper.a
  COPY Release/scrypt_wrapper.a
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.2.0/lib/crypto/crypto_scrypt.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.2.0/lib/crypto/crypto_scrypt_smix.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.2.0/libcperciva/util/warnp.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.2.0/libcperciva/alg/sha256.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.2.0/libcperciva/util/insecure_memzero.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.2.0/lib/scryptenc/scryptenc_cpuperf.o
  AR(target) Release/obj.target/scrypt_lib.a
  COPY Release/scrypt_lib.a
  CXX(target) Release/obj.target/scrypt/src/node-boilerplate/scrypt_common.o
  CXX(target) Release/obj.target/scrypt/src/node-boilerplate/scrypt_params_async.o
In file included from ../src/node-boilerplate/inc/scrypt_async.h:28:0,
                 from ../src/node-boilerplate/inc/scrypt_params_async.h:28,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_common.h: In constructor ‘NodeScrypt::Params::Params(const v8::Local<v8::Object>&)’:
../src/node-boilerplate/inc/scrypt_common.h:39:48: warning: ‘v8::Local<v8::Value> v8::Object::Get(v8::Local<v8::Value>)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       N(obj->Get(Nan::New("N").ToLocalChecked())->Uint32Value()),
                                                ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:3553:51: note: declared here
   V8_DEPRECATED("Use maybe version", Local<Value> Get(Local<Value> key));
                                                   ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/node-boilerplate/inc/scrypt_async.h:28:0,
                 from ../src/node-boilerplate/inc/scrypt_params_async.h:28,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_common.h:39:63: error: no matching function for call to ‘v8::Value::Uint32Value()’
       N(obj->Get(Nan::New("N").ToLocalChecked())->Uint32Value()),
                                                               ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2707:41: note: candidate: v8::Maybe<unsigned int> v8::Value::Uint32Value(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<uint32_t> Uint32Value(
                                         ^~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2707:41: note:   candidate expects 1 argument, 0 provided
In file included from ../src/node-boilerplate/inc/scrypt_async.h:28:0,
                 from ../src/node-boilerplate/inc/scrypt_params_async.h:28,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_common.h:40:48: warning: ‘v8::Local<v8::Value> v8::Object::Get(v8::Local<v8::Value>)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       r(obj->Get(Nan::New("r").ToLocalChecked())->Uint32Value()),
                                                ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:3553:51: note: declared here
   V8_DEPRECATED("Use maybe version", Local<Value> Get(Local<Value> key));
                                                   ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/node-boilerplate/inc/scrypt_async.h:28:0,
                 from ../src/node-boilerplate/inc/scrypt_params_async.h:28,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_common.h:40:63: error: no matching function for call to ‘v8::Value::Uint32Value()’
       r(obj->Get(Nan::New("r").ToLocalChecked())->Uint32Value()),
                                                               ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2707:41: note: candidate: v8::Maybe<unsigned int> v8::Value::Uint32Value(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<uint32_t> Uint32Value(
                                         ^~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2707:41: note:   candidate expects 1 argument, 0 provided
In file included from ../src/node-boilerplate/inc/scrypt_async.h:28:0,
                 from ../src/node-boilerplate/inc/scrypt_params_async.h:28,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_common.h:41:48: warning: ‘v8::Local<v8::Value> v8::Object::Get(v8::Local<v8::Value>)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       p(obj->Get(Nan::New("p").ToLocalChecked())->Uint32Value()) {}
                                                ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:3553:51: note: declared here
   V8_DEPRECATED("Use maybe version", Local<Value> Get(Local<Value> key));
                                                   ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/node-boilerplate/inc/scrypt_async.h:28:0,
                 from ../src/node-boilerplate/inc/scrypt_params_async.h:28,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_common.h:41:63: error: no matching function for call to ‘v8::Value::Uint32Value()’
       p(obj->Get(Nan::New("p").ToLocalChecked())->Uint32Value()) {}
                                                               ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2707:41: note: candidate: v8::Maybe<unsigned int> v8::Value::Uint32Value(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<uint32_t> Uint32Value(
                                         ^~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2707:41: note:   candidate expects 1 argument, 0 provided
In file included from ../src/node-boilerplate/inc/scrypt_params_async.h:28:0,
                 from ../src/node-boilerplate/scrypt_params_async.cc:4:
../src/node-boilerplate/inc/scrypt_async.h: In member function ‘virtual void ScryptAsyncWorker::HandleErrorCallback()’:
../src/node-boilerplate/inc/scrypt_async.h:53:29: warning: ‘v8::Local<v8::Value> Nan::Callback::Call(int, v8::Local<v8::Value>*) const’ is deprecated [-Wdeprecated-declarations]
       callback->Call(1, argv);
                             ^
In file included from ../src/node-boilerplate/scrypt_params_async.cc:1:0:
../../nan/nan.h:1742:3: note: declared here
   Call(int argc, v8::Local<v8::Value> argv[]) const {
   ^~~~
In file included from ../src/node-boilerplate/scrypt_params_async.cc:4:0:
../src/node-boilerplate/inc/scrypt_params_async.h: In constructor ‘ScryptParamsAsyncWorker::ScryptParamsAsyncWorker(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/node-boilerplate/inc/scrypt_params_async.h:35:36: error: no matching function for call to ‘v8::Value::NumberValue()’
       maxtime(info[0]->NumberValue()),
                                    ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2704:39: note: candidate: v8::Maybe<double> v8::Value::NumberValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<double> NumberValue(Local<Context> context) const;
                                       ^~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2704:39: note:   candidate expects 1 argument, 0 provided
In file included from ../src/node-boilerplate/scrypt_params_async.cc:4:0:
../src/node-boilerplate/inc/scrypt_params_async.h:36:39: error: no matching function for call to ‘v8::Value::NumberValue()’
       maxmemfrac(info[1]->NumberValue()),
                                       ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2704:39: note: candidate: v8::Maybe<double> v8::Value::NumberValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<double> NumberValue(Local<Context> context) const;
                                       ^~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2704:39: note:   candidate expects 1 argument, 0 provided
In file included from ../src/node-boilerplate/scrypt_params_async.cc:4:0:
../src/node-boilerplate/inc/scrypt_params_async.h:37:36: error: no matching function for call to ‘v8::Value::IntegerValue()’
       maxmem(info[2]->IntegerValue()),
                                    ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note: candidate: v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
                                        ^~~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
In file included from ../src/node-boilerplate/scrypt_params_async.cc:4:0:
../src/node-boilerplate/inc/scrypt_params_async.h:38:39: error: no matching function for call to ‘v8::Value::IntegerValue()’
       osfreemem(info[3]->IntegerValue())
                                       ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67:0,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note: candidate: v8::Maybe<long int> v8::Value::IntegerValue(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT Maybe<int64_t> IntegerValue(
                                        ^~~~~~~~~~~~
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:2705:40: note:   candidate expects 1 argument, 0 provided
../src/node-boilerplate/scrypt_params_async.cc: In member function ‘virtual void ScryptParamsAsyncWorker::HandleOKCallback()’:
../src/node-boilerplate/scrypt_params_async.cc:23:67: warning: ‘bool v8::Object::Set(v8::Local<v8::Value>, v8::Local<v8::Value>)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
   obj->Set(Nan::New("N").ToLocalChecked(), Nan::New<Integer>(logN));
                                                                   ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:3499:22: note: declared here
                 bool Set(Local<Value> key, Local<Value> value));
                      ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
../src/node-boilerplate/scrypt_params_async.cc:24:64: warning: ‘bool v8::Object::Set(v8::Local<v8::Value>, v8::Local<v8::Value>)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
   obj->Set(Nan::New("r").ToLocalChecked(), Nan::New<Integer>(r));
                                                                ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:3499:22: note: declared here
                 bool Set(Local<Value> key, Local<Value> value));
                      ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
../src/node-boilerplate/scrypt_params_async.cc:25:64: warning: ‘bool v8::Object::Set(v8::Local<v8::Value>, v8::Local<v8::Value>)’ is deprecated: Use maybe version [-Wdeprecated-declarations]
   obj->Set(Nan::New("p").ToLocalChecked(), Nan::New<Integer>(p));
                                                                ^
In file included from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8-internal.h:14:0,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:27,
                 from /home/rbarriuso/.cache/node-gyp/12.18.3/include/node/node.h:67,
                 from ../../nan/nan.h:56,
                 from ../src/node-boilerplate/scrypt_params_async.cc:1:
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8.h:3499:22: note: declared here
                 bool Set(Local<Value> key, Local<Value> value));
                      ^
/home/rbarriuso/.cache/node-gyp/12.18.3/include/node/v8config.h:328:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
../src/node-boilerplate/scrypt_params_async.cc:32:25: warning: ‘v8::Local<v8::Value> Nan::Callback::Call(int, v8::Local<v8::Value>*) const’ is deprecated [-Wdeprecated-declarations]
   callback->Call(2, argv);
                         ^
In file included from ../src/node-boilerplate/scrypt_params_async.cc:1:0:
../../nan/nan.h:1742:3: note: declared here
   Call(int argc, v8::Local<v8::Value> argv[]) const {
   ^~~~
scrypt.target.mk:131: fallo en las instrucciones para el objetivo 'Release/obj.target/scrypt/src/node-boilerplate/scrypt_params_async.o'
make: *** [Release/obj.target/scrypt/src/node-boilerplate/scrypt_params_async.o] Error 1
make: se sale del directorio '/home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/scrypt/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/rbarriuso/.nvm/versions/node/v12.18.3/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Linux 4.15.0-121-generic
gyp ERR! command "/home/rbarriuso/.nvm/versions/node/v12.18.3/bin/node" "/home/rbarriuso/.nvm/versions/node/v12.18.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/rbarriuso/Datos/Proyectos/Blockchain/alastria-identity-lib/node_modules/scrypt
gyp ERR! node -v v12.18.3
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! scrypt@6.0.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the scrypt@6.0.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rbarriuso/.npm/_logs/2020-10-16T16_26_02_396Z-debug.log

```

Due to this issue it's not possible to use Node 12 with https://github.com/alastria/alastria-wallet

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "chai-as-promised": "^7.1.1",
     "ethereumjs-tx": "^1.3.3",
     "jsontokens": "2.0.2",
-    "keythereum": "1.0.4",
+    "keythereum": "^1.2.0",
     "npm-run-all": "4.1.5",
     "rimraf": "2.6.3",
     "typescript": "3.5.2",

--- a/package.json
+++ b/package.json
@@ -43,8 +43,7 @@
     "npm-run-all": "4.1.5",
     "rimraf": "2.6.3",
     "typescript": "3.5.2",
-    "web3": "1.2.1",
-    "web3-utils": "^1.0.0"
+    "web3": "1.2.1"
   },
   "devDependencies": {
     "@types/chai": "^4.2.11",

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
     "rimraf": "2.6.3",
     "typescript": "3.5.2",
     "web3": "1.2.1",
-    "web3-utils": "1.0.0-beta.55"
+    "web3-utils": "^1.0.0"
   },
   "devDependencies": {
     "@types/chai": "^4.2.11",

